meetings/2016-12-07.md

```diff
@@ -0,0 +1,313 @@
+* Jenn Turner @renrutnnej (observer/Node.js Foundation):
+ * No update, just observing.
+* Seth Thompson @s3ththompson (observer/Google):
+  * At Node Interactive & vm neutrality summit
+  * Circulating the discussions with the V8 team.
+* Michaël Zasso @targos (observer):
+  * Reviewed PRs and commented on issues.
+* Bradley Farias @Bmeck (TC39 / observer):
+  * Met w/ v8 and google about various module topics
+* Rich Trott @Trott (CTC):
+* James M Snell @jasnell (CTC):
+  * Not present during the main part of the call, but…
+  * Node.js Interactive last week
+  * Landing a few PRs this week
+  * Continued work on HTTP/2 implementation
+
+---
+
+## Agenda
+
+### nodejs/node
+
+* deps: upgrade npm to 4.0.2 [#9848](https://github.com/nodejs/node/pull/9848)
+
+### nodejs/node-eps
+
+* Initial version of eps for ABI-Stable-Module-API
+  [#20](https://github.com/nodejs/node-eps/pull/20)
+
+### nodejs/CTC
+
+* Moving `node-inspect` into core [#40](https://github.com/nodejs/CTC/issues/40)
+
+---
+
+## Previous Meeting Review
+
+### nodejs/node
+
+* Revert "buffer: runtime deprecation of calling Buffer without new"
+  [#9529](https://github.com/nodejs/node/pull/9529)
+
+---
+
+## Minutes
+
+### deps: upgrade npm to 4.0.2 [#9848](https://github.com/nodejs/node/pull/9848)
+
+@Fishrock123: New major, need to consider impact and when to land. May not need
+to be in a Node.js major change.
+
+@mhdawson: Is it breaking?
+
+@Fishrock123: Yes, but really obscure edge cases AFAIK.
+
+@addaleax: Yes, it’s technically breaking but I don’t think it will break
+anyone’s code.
+
+@evanlucas: Which version would we land on?
+
+@addaleax: npm plans to get 5.x release out in first quarter of 2017, so we may
+be able to skip 4.x entirely.
+
+@Fishrock123: So question is whether to land in current 7.x branches.
+
+@evanlucas: I think they may have backported changes to version 3, but if it’s
+not being maintained we may have to go to version 4.
+
+@Fishrock123: Bug fixing maintenance will continue, but no new features.
+
+@mhdawson: Question is what’s the advantage of being part of a 7.x release, as
+opposed to waiting for 8.x.
+
+@Fishrock123 @addaleax: it should not be ported back to 6.x.
+
+@evanlucas: Should we ship with both 3 and 4?
+
+People can update npm themselves.
+
+@addaleax: I’ve heard that npm doesn’t feel strongly about getting it in 7.x,
+but would like everyone to move to 4.
+
+@addaleax: I’m okay to land in 7.x. Very few cases where code will be broken.
+
+**Next steps**:
+
+* Take back to GitHub for more discussion.
+
+
+---
+
+### Moving `node-inspect` into core [#40](https://github.com/nodejs/CTC/issues/40)
+
+See also
+[nodejs/diagnostics#67](https://github.com/nodejs/diagnostics/issues/67).
+
+@Trott: Old debugger protocol and CLI debugger is going away.
+
+@ofrobots: Deprecating old CLI debugger, can build a new CLI debugger based on
+new inspector protocol.
+
+PR for removal of V8 Debug API: <https://github.com/nodejs/node/issues/9789>
```

Comment:
https://github.com/nodejs/node/issues/9789 is not a pull request, it's an issue.